src/transport/SecureSessionMgr.cpp

```diff
@@ -173,24 +173,25 @@ CHIP_ERROR SecureSessionMgrBase::SendMessage(PayloadHeader & payloadHeader, Node
     return err;
 }
 
-CHIP_ERROR SecureSessionMgrBase::NewPairing(const Optional<Transport::PeerAddress> & peerAddr, SecurePairingSession * pairing)
+CHIP_ERROR SecureSessionMgrBase::NewPairing(const Optional<Transport::PeerAddress> & peerAddr, NodeId peerNodeId,
+                                            SecurePairingSession * pairing)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
 
-    Optional<NodeId> peerNodeId = Optional<NodeId>::Value(pairing->GetPeerNodeId());
```

Comment:
It would be great if the commit message explained why we can't use the pairing's `GetPeerNodeId` here and have to get the peer node id passed in as an argument instead.

Reply:
done

Reply:
The commit message still looks pretty empty to me....

Maybe a bunch of the information from the initial PR comment should move into the commit message?

Reply:
It should be updated now..

Reply:
Yep, looks great!